doc/examples/xx_applications/plot_haar_extraction_selection_classification.py

```diff
@@ -74,9 +74,9 @@ def extract_feature_image(img, feature_type, feature_coord=None):
 # the computation step
 X = delayed(extract_feature_image(img, feature_types)
             for img in images)
-# Compute the result using the "multiprocessing" dask backend
+# Compute the result using the "processes" dask backend
 t_start = time()
-X = np.array(X.compute(get=multiprocessing.get))
```

Comment:
I think you should also remove the released imports.

Reply:
Good point. Fixed. Thank you.